NAMESPACE

```diff
@@ -1 +1 @@
-
+exportPattern("^[[:alpha:]]+")
```

Comment:
How come your function is not in here? Because you do say "@export" in the function so I think it's supposed to be added to the namespace then?

Reply:
I'm talking about "create_skeleton"